test/python/transpiler/test_preset_passmanagers.py

```diff
@@ -97,17 +107,24 @@ def test_initial_layout_2(self):
         See: https://github.com/Qiskit/qiskit-terra/issues/2532
         """
         # build a circuit which works as-is on the coupling map, using the initial layout
-        qr = QuantumRegister(5)
+        qr = QuantumRegister(5, 'q')
         cr = ClassicalRegister(2)
+        ancilla = QuantumRegister(9, 'ancilla')
         qc = QuantumCircuit(qr, cr)
         qc.cx(qr[2], qr[4])
         initial_layout = {qr[2]: 11, qr[4]: 3,  # map to [11, 3] connection
                           qr[0]: 1, qr[1]: 5, qr[3]: 9}
+        final_layout = {0: ancilla[0], 1: qr[0], 2: ancilla[1], 3: qr[4], 4: ancilla[2], 5: qr[1],
+                        6: ancilla[3], 7: ancilla[4], 8: ancilla[5], 9: qr[3], 10: ancilla[6],
+                        11: qr[2], 12: ancilla[7], 13: ancilla[8]}
         backend = FakeMelbourne()
 
         for optimization_level in range(4):
             qc_b = transpile(qc, backend, initial_layout=initial_layout,
                              optimization_level=optimization_level)
+            for wire in range(14):
+                with self.subTest(level=optimization_level, wire=wire):
+                    self.assertEqual(qc_b.layout[wire], final_layout[wire])
```

Comment:
this doesn't really seem like a subtest. you just want to compare if two dictionaries are equal, not create a new test for each single wire!

Reply:
Indeed. I was combinant during the creation of the test. Removed in d9aabcc3